src/main/webapp/user-private-comments.jsp

```diff
@@ -14,27 +13,53 @@
 %>
 
 <%--HTML--%>
-<ul>
-  <%if (currUser) {%>
-    <h1 id="pc-header">Your Private Task Comments:</h1>
-    <%for (TaskData task : tasks) {%>
-      <li class="task-with-pc">
-        <button class="inline deep-button" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
-          <%=task.getName()%>
-        </button>
-        <%request.setAttribute("task", task);%>
-        <%request.setAttribute("clickable", false);%>
-        <jsp:include page="task-status-checkmark.jsp"/>
-        <p class="inline"><%=task.getDescription()%></p>
-        <form action="/user-profile" method="POST">
-          <input type="hidden" name="taskID" value="<%=task.getTaskID()%>">
-          <br>
-          <textarea style="width:300px; height:100px;" type="text" name="message-<%=task.getTaskID()%>" rows="6" cols="50"><%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%></textarea>
-          <br>
-          <button class="deep-button" style="font-size: 12px;" type="submit" class="deep-button">Update Comment</button>
-        </form>
-        <br><br>
-      </li>
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@editorjs/header@latest"></script>
+</head>   
+  <ul>
+    <%if (currUser) {%>
+      <h1 id="pc-header">Your Private Task Comments:</h1>
+      <%for (TaskData task : tasks) {%>
+        
+        <script>
+          function init() {
+            const editor = new EditorJS({
+            holderId : 'box',
+            autofocus: true,
+            data: '<%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%>',
+            tools: {
+              header: Header,
+             },
+            });
+          }
+          function save() {
+            editor.save().then((outputData) => {
```

Comment:
would "outputData" be what I want to pass to an <input> to save in Datastore?

Reply:
If so, can I just somehow reformat it to java?

Reply:
yep outputData would contain the box data that you'd save into Datastore.  from here, you can then push that into an input right before the form is submitted:
```
document.getElementById('private-comment-12345').value(outputData);
```